db/migrate/20220128111006_add_foreign_key_from_submission_to_admin_user.rb

```diff
@@ -0,0 +1,41 @@
+class AddForeignKeyFromSubmissionToAdminUser < ActiveRecord::Migration[6.1]
+  def up
+    add_column :submissions, :created_by_id, :bigint
```

Comment:
**suggestion:** We may want to take this opportunity to rename this column so that its intention is clear. At the moment it's not clear why some submissions have a `created_by` and some don't, because all submissions were created by _someone_.

What do you think about something simple like `admin_id`?

Reply:
Updated